draft-ietf-ivy-network-inventory-yang.md

```diff
@@ -164,55 +164,58 @@ The YANG data model defined in this document conforms to the Network Management
 
 ## Terminology
 
-  The following terms are defined in {{!RFC7950}} and are not
-  redefined here:
+The following terms are defined in {{!RFC7950}} and are not redefined here:
+- client
+- server
+- augment
+- data model
+- data node
 
-  *  client
+The following terms are defined in {{!RFC6241}} and are not redefined here:
+- configuration data
+- state data
 
-  *  server
+The following terms are defined in {{IANA_YANG}} and are not redefined here:
+- backplane
```

Comment:
All these terms are not formally defined in 8348. Maybe change to "used" or "introduced".

Reply:
See: https://github.com/ietf-ivy-wg/network-inventory-yang/pull/75/files#r1886557028